roles/kubernetes/control-plane/defaults/main/main.yml

```diff
@@ -162,9 +206,10 @@ kube_apiserver_admission_plugins_podnodeselector_default_node_selector: ""
 # kube_oidc_client_id: kubernetes
 ## Optional settings for OIDC
 # kube_oidc_username_claim: sub
-# kube_oidc_username_prefix: 'oidc:'
+# kube_oidc_username_prefix: "oidc:"
 # kube_oidc_groups_claim: groups
-# kube_oidc_groups_prefix: 'oidc:'
+# kube_oidc_groups_prefix: "oidc:"
+# kube_oidc_uid_claim: "sub"
```

Comment:
Any reason for the change here ? If it's not related let's drop it from this PR.